<launch>
	<arg name="veh" doc="Name of vehicle" />
    <arg name="local" default="true" doc="true for running everything you possibly can on laptop. false for everything running on vehicle."/>
    <arg name="config" default="baseline" doc="Specify a config."/>
    <arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman." />

	<remap from="vehicle_filter_node/corners" to="vehicle_filter_test_node/corners" />
	<remap from="vehicle_filter_node/camera_info" to="vehicle_filter_test_node/camera_info" />
	<include file="$(find vehicle_detection)/launch/vehicle_filter_node.launch">
		<arg name="veh" value="$(arg veh)"/>
		<arg name="local" value="true" />
	</include>

	<remap from="vehicle_filter_test_node/pose" to="vehicle_filter_node/pose" />
	<group ns="$(arg veh)">
        <node name="vehicle_filter_test_node" pkg="vehicle_detection" type="vehicle_filter_node_test.py" output="screen" />
	</group>
	
</launch>

